app/src/utils/constants/housing.ts

```diff
@@ -94,4 +94,10 @@ export const PROJECT_LOCATION_LIST = [ProjectLocation.LOCATION_COORDINATES, Proj
 
 export const QUEUE_PRIORITY = [1, 2, 3];
 
-export const SUBMISSION_TYPE_LIST = [SubmissionType.GUIDANCE, SubmissionType.INAPPLICABLE];
+export const SUBMISSION_TYPE_LIST = [
+  SubmissionType.ESCALATION,
+  SubmissionType.GENERAL_ENQUIRY,
+  SubmissionType.GUIDANCE,
+  SubmissionType.INAPPLICABLE,
+  SubmissionType.STATUS_REQUEST
+];
```

Comment:
This is mixing Submission types with Enquiry types. If you need both lists you need to create a union of the two.

Reply:
Git blame shows that I somehow overwrote the correct version with the incorrect version. No idea how that happened because I don't need to do this. Reverting.